### PR TITLE
Tooltips

### PR DESCRIPTION
**Pull request by nvdaes**

### CODE REVIEW

**Comment by nvdaes** [comment]
Fixes issue #2321

**Comment by NickColley** [comment]
Adding aria-expands to this seems sensible, thank you for doing this. It seems though that your pull request also includes spanish translations, when you get some time would you be ok to split any unrelated contribution outside of this pull request? This way we can review and merge smaller contributions in isolation quicker, cheers.

**Comment by nvdaes** [comment]
> It seems though that your pull request also includes spanish translations...

Oh, sorry so much. This is a mistake. I work with wsl and standar Windows in different folders, and I'm working in a PR for spanish translation, so I've mixed things accidentally. Thanks for pointing this out. In fact, I realised about this inconvenient with tooltips while testing the spanish translation.
I'll split this.

**Comment by nvdaes** [comment]
I think that this is ready for review. Thanks.

File: src/routes/_components/Tooltip.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
 -->
 <span class="tooltip-button"
       aria-describedby="tooltip-{id}"
+            aria-expanded={shown}
```
Comment (nolanlawson) [inline src/routes/_components/Tooltip.html:6]:
```suggestion
      aria-expanded={shown}
```

Nit: alignment.

**Review by nolanlawson — APPROVED** [review]

**Comment by nolanlawson** [comment]
Thank you!

**Comment by nvdaes** [comment]
Hank you for the alignment. >I won"t make further changes after you approved pull requests, so you can merge them when you want.

Enviado desde mi iPhone

**Pull request merged**